File: JS/URI/1050.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n');

const ddd = parseInt(lines.shift());

switch(ddd)
{
    case 11:
        console.log("Sao Paulo");
        break;

    case 19:
        console.log("Campinas");
        break;

    case 21:
        console.log("Rio de Janeiro");
        break;

    case 27:
        console.log("Vitoria");
        break;

    case 32:
        console.log("Juiz de Fora");
        break;

    case 31:
        console.log("Belo Horizonte");
        break;

    case 61:
        console.log("Brasilia");
        break;

    case 71:
        console.log("Salvador");
        break;

    default:
        console.log("DDD nao cadastrado");
        break;
}

File: C++/1050.cpp
#include<iostream>
#include <iostream>
#include <string>
#include <limits>
#include <cmath>
#include <iomanip>

using namespace std;

int main(){

    int ddd;

    cin >> ddd;

    switch(ddd)
    {
        case 11:
            cout << "Sao Paulo\n";
            break;

        case 19:
            cout << "Campinas\n";
            break;

        case 21:
            cout << "Rio de Janeiro\n";
            break;

        case 27:
            cout << "Vitoria\n";
            break;

        case 32:
            cout << "Juiz de Fora\n";
            break;

        case 31:
            cout << "Belo Horizonte\n";
            break;

        case 61:
            cout << "Brasilia\n";
            break;

        case 71:
            cout << "Salvador\n";
            break;

        default:
            cout << "DDD nao cadastrado\n";
            break;
    }

}

